docs/recipes/install/common/install_slurm.tex

```diff
@@ -36,10 +36,10 @@
   for compute nodes under its control. In particular, three configuration
   parameters combine to define consumable compute resources: {\em Sockets},
   {\em CoresPerSocket}, and {\em ThreadsPerCore}. The default configuration
-  file provided via \OHPC{} assumes dual-socket, 8 cores per socket, and two
+  file provided via \OHPC{} assumes the nodes are named c1-c4 and are dual-socket, 8 cores per socket, and two
   threads per core for this 4-node example. If this does not reflect your local
   hardware, please update the configuration file at
-  \path{/etc/slurm/slurm.conf} accordingly to match your particular hardware.
+  \path{/etc/slurm/slurm.conf} accordingly to match your nodes names and particular hardware. Be sure to run \texttt{scontrol reconfigure} to notify SLURM of the changes. 
```

Comment:
nit: the line above is too long compared to the rest. Consider moving the new sentence on its own new line.

Reply:
Paragraph wrapped with new commit.